src/react/components/pages/modelCompose/modelCompose.tsx

```diff
@@ -589,7 +596,8 @@ export default class ModelComposePage extends React.Component<IModelComposePageP
     }
 
     private passSelectedItems = (Items: any[]) => {
-        this.selectedItems = Items;
+        this.cannotBeIncludedItems = Items.filter((item: IModel) => item.status !== "ready");
```

Comment:
put "ready" in constants.ts so we can find all references more easily.

Reply:
done